src/core/reducers/ui.ts

```diff
@@ -18,12 +18,16 @@ export default function updateUi(state: State = {}, { type, payload, meta = <Act
   }
 }
 
-export const createComponentState = (state: State, { tagName, id, state: tagState }: Actions.Payload.Component.State) =>
+// tslint:disable-next-line max-line-length
+export const createComponentState = (state: State, { tagName, id, state: tagState, persist }: Actions.Payload.Component.State) =>
   ({
     ...state,
     [tagName]: {
       ...state[tagName],
-      [id]: tagState
+      [id]: {
```

Comment:
why the change?

Reply:
Wanted to add a "persist" flag so that info could persist over time